dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/SearchJobTask.java

```diff
@@ -63,24 +71,12 @@ public SearchJobTask(final List<Bucket> searchableBuckets, final String searchTe
             return list;
         } catch (final Exception e) {
             LOG.error("Search failed", e);
-            printLog(StringBuilderUtil.searchFailedMessage().toString() + e, LogType.ERROR);
+            loggingService.logMessage(StringBuilderUtil.searchFailedMessage().toString() + e, LogType.ERROR);
```

Comment:
Let's clean this up a bit.  The `searchFailedMessage()` call returns a StringBuilder, which we immediately turn into a String and then concatenate the exception's `toString()` method to it.  Instead, after we get the StringBuilder from `searchFailedMessage()`, append the exception to it, then call `toString()`